driver-examples/tycho/README.md

```diff
@@ -0,0 +1,47 @@
+# Java Driver examples
+
+This project demonstrates how to integrate the Java Driver in an Eclipse RCP application
+built with Maven and Tycho.
+
+## Packaging the Java Driver in an Eclipse RCP application
+
+The Java Driver is built and distributed as an OSGi bundle, but integrating it in an RCP application is not trivial.
+
+If you are using a "manifest-first" approach and your project/company benefits from a dedicated P2 update site, it is best to place the driver bundle, 
+along with its dependencies, in your update site, and have Tycho resolve the dependencies from there.
+
+Alternatively, it is also possible to "wrap" the Java Driver jar, along with its dependencies,
+in a plugin submodule in your application, and have Tycho build it along with your application.
+
+The current project uses a third option: a "pom-first" approach, where the driver dependencies are pulled directly from Maven repositories.
+For this to work, Tycho has been configured to consider pom dependencies when resolving dependencies.
+
+## Installing/Running with Tycho
+
+To build the project and run the tests, simply run
+
+    mvn integration-test
+    
+Please note: currently the tests require an instance of Cassandra running on localhost:9042.
```

Comment:
It was hard to pull the test-jar dependency. I had to completely refactor the way I create my P2 installation. That part is now working.
However, CCMBridge will still be hard to use, because the tests bundle is a fragment of the mailbox bundle, which means the the mailbox Activator class is called _before_ the test is even started. In this situation, it is hard to start ccm before that. I still need to figure out a solution for that.